### DISCUSSION TITLE
leverage

### DISCUSSION THREAD

**Discussion by giscus** [mrocklin/mrocklin.github.io#22] [Giscus]
# leverage

Every human being has a basic instinct to help each other out – Mark Watney, The Martian Leverage and the loss of ego: Positions of leverage feel great, but result in self-sacrifice and burnout. Wh...

https://matthewrocklin.com/leverage.html

**Comment by tkoyama010** [comment]
The way we deal with our work is important for everyone, including me. Your words have encouraged me a lot. I will follow my heart's voice. Thank you for this post :)

**Comment by edouardfouche** [comment]
Thank you Matthew for the inspiring post and for being so open. I do not know if that is fair to call that behavior "selfish". Afterall, if you are taking self-care measures, so that you can *sustainably* support your colleagues (as opposed to support them for a while until you burn out), then that is not a selfish action. As most for-profit companies try to squeeze out every drop of productivity from their employees (that is just in their DNA), calling this kind of action selfish may just be a symptom of short-sightedness, based on the largely false assumption that burned out employees are easy and cheap to replace.
